src/pages/HomePage.tsx

```diff
@@ -0,0 +1,5 @@
+import { FC } from 'react';
+
+export const HomePage: FC = () => (
+  <h1 className="title">Home page</h1>
+);
```

Comment:
Use a container in similar cases

```suggestion
export const HomePage: FC = () => (
  <div className="container">
    <h1 className="title">Home page</h1>
  </div>
);
```